relay-server/src/services/processor/span.rs

```diff
@@ -16,16 +22,73 @@ pub use processing::*;
 
 pub fn filter(state: &mut ProcessEnvelopeState<SpanGroup>) {
     let disabled = state.should_filter(Feature::StandaloneSpanIngestion);
+    let otel_disabled = state.should_filter(Feature::OtelEndpoint);
```

Comment:
We should also check this in the fast path (aka `check_envelope`).

Reply:
We do, via https://github.com/getsentry/relay/blob/63a88f964098463277c70f0583446af8818d891c/relay-server/src/services/projects/project/info.rs#L127-L134

Not sure if that check should use `should_filter` instead of `has_feature`.